backend/pong/match/tests/test_geometry_utils.py

```
@@ -0,0 +1,161 @@
+        result = check_opposite_sides(segment_vec, point_vec1, point_vec2)
+        self.assertFalse(result)
+
+    def test_is_internally_divided_true(self) -> None:
+        """
+        点が線分を内分する場合のテスト。
+        点pが線分v1-v2上にあるため、結果がTrueであることを確認します。
+        """
+        v1 = np.array([0, 0])
+        v2 = np.array([2, 2])
+        p = np.array([1, 1])
+        result = is_internally_dlvided(v1, v2, p)
```

Ruff (0.8.2)

Undefined name `is_internally_dlvided`

(F821)